sw/airborne/modules/computer_vision/lib/vision/image.c

```diff
@@ -27,6 +27,7 @@
 #include "image.h"
 #include <stdlib.h>
 #include <string.h>
+#include "lucas_kanade.h"
```

Comment:
This dependency is a little strange, in principle the image.c/h should not be dependent on a specific optical flow file. That said, I also don't think the image_show_flow should be in this file either as it is specific to optical flow and not generic images. In principle, the draw vector should be here and the image_show_flow should be wherever the flow functions are implemented.

I won't ask you to change this here as you didn't introduce the core problem and I know you don't have time to fix it but consider this in the future.